### ISSUE TITLE
BrowserBox Tor Hidden Service

### ISSUE THREAD

**Issue by gunthejump** [gunthejump/BrowserBox#1]
# Set Up a Tor Hidden Service

Before starting, [fork](../fork) or [generate](../generate) this repo to your account (not for orgs).

To begin, click **Submit New Issue**. Instructions will appear in the comments.

Your BrowserBox hidden service on Tor will be ready soon. Watch for prompts in the comments for guidance.

*Note: This service activates only when you open or reopen the issue. It won't consume your GitHub Actions minutes unnecessarily. Only you will be able to run this job by opening this issue, nobody else's issue will be able to trigger it. Edit `userjob.yaml` to change this.*

After clicking **Submit New Issue**, you'll receive your .onion link in the comments in a few minutes! :laughing:

---

*BrowserBox by [DOSYAGO](https://dosyago.com)*

**Support Tor**: [Donate to Tor](https://donate.torproject.org)
